frontend/src/App.jsx

```diff
@@ -12,9 +13,9 @@ function App() {
       <Route path="/" element={<Home />} />
       <Route path="/profile" element={<Profile />} />
       <Route path="/profile/:profileId" element={<Profile />} />
+      <Route path="/register" element={<Register />} /> 
+      <Navbar />
```

Comment:
pourquoi navbar et dans le router et il manque une div fermante

Reply:
Tired, sorry for that